internal/pkg/manifest/worker_svc.go

```diff
@@ -206,12 +208,39 @@ func NewWorkerService(props WorkerServiceProps) *WorkerService {
 		svc.WorkerServiceConfig.TaskConfig.CPU = aws.Int(MinWindowsTaskCPU)
 		svc.WorkerServiceConfig.TaskConfig.Memory = aws.Int(MinWindowsTaskMemory)
 	}
+	if len(props.Topics) > 0 {
+		if aws.BoolValue(FIFOEnabled(props.Topics)) {
+			svc.WorkerServiceConfig.Subscribe.Queue.FIFO.Enable = aws.Bool(true)
+		}
+	}
 	svc.WorkerServiceConfig.Subscribe.Topics = props.Topics
 	svc.WorkerServiceConfig.Platform = props.Platform
 	svc.parser = template.New()
 	return svc
 }
 
+func FIFOEnabled(topics []TopicSubscription) *bool {
```

Comment:
Can this be private? 🥺

Reply:
Right now this function sounds like it doesn't have any side-effects (like changing the underlying `TopicSubscriptions`), but it indeed does mutate the data.

What do you think of a structure like this:
```go
func setSubscriptionQueueDefaults(topics []TopicSubscription, eventsQueue *SQSQueue) {
  //...
  if !isFIFOEnabled {
     return
  }
  eventsQueue.FIFO.Enable = aws.Bool(true)
  for idx, topic := range topics {
	if strings.HasSuffix(aws.StringValue(topic.Name), ".fifo") {
		topics[idx].Name = aws.String(strings.TrimSuffix(aws.StringValue(topic.Name), ".fifo"))
	} else {
		topics[idx].Queue.Enabled = aws.Bool(true)
	}
  }
}
```
and then in the constructor we can do:
```go
if len(props.Topic) > 0 {
   setSubscriptionQueueDefaults(props.Topics, &svc.WorkerServiceConfig.Subscribe.Queue)
}
```